### PR TITLE
clarify Azure permissions

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2633 
Review deadline:

## Page previews
https://deploy-preview-13--rp-cloud.netlify.app/redpanda-cloud/security/authorization/cloud-iam-policies-azure/

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Review by chapmanc — APPROVED** [review]
Looks good to me.

File: modules/security/partials/iam-policies.adoc [hunk 1]
```diff
@@ -747,19 +747,15 @@ endif::[]
 
 
 ifdef::env-azure[]
-Azure IAM permissions are scoped to resource groups. For example:
+Azure RBAC is scoped to resource groups. For example:
```
Comment (Feediver1) [inline modules/security/partials/iam-policies.adoc:750]:
```suggestion
Azure RBAC (role-based access control) is scoped to resource groups. For example:
```

Comment (Feediver1) [reply modules/security/partials/iam-policies.adoc:750]: Given that it is the first mention of RBAC in this doc.

**Review by Feediver1 — APPROVED** [review]
one suggestion

**Pull request merged**